etc/emqx.conf

```diff
@@ -458,7 +458,35 @@ log.file = emqx.log
 
 ## Log formatter
 ## Value: text | json
-#log.formatter = text
+log.formatter = text
+
+## Text logger, date format.
+##
+## Value: rfc3339 | FormatString
+##   FormatString:
+##     %Y: year
+##     %m: month
+##     %d: day
+##     %H: hour
+##     %M: minute
+##     %S: second
+##     %6N: microseconds
+##     %3N: milliseconds
+##     %z: timezone, [+|-]HH:MM or [+|-]HH:MM:SS. Depends on `log.formatter.text.date.timezone`
```

Comment:
We can support `%z`, `%:z`,  `%::z` just like what the GNU `date` command supports.